src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,30 +39,35 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${selectedTodo.id}`}
             </div>
 
-            {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-            <button
-              type="button"
-              className="delete"
-              data-cy="modal-close"
-            />
+            {selectedTodo && (
+              // eslint-disable-next-line jsx-a11y/control-has-associated-label
+              <button
+                type="button"
+                className="delete"
+                data-cy="modal-close"
+                onClick={() => setSelectedTodo(null)}
+              />
+            )}
           </header>
 
           <div className="modal-card-body">
             <p className="block" data-cy="modal-title">
-              quis ut nam facilis et officia qui
+              {selectedTodo.title}
             </p>
 
             <p className="block" data-cy="modal-user">
               {/* <strong className="has-text-success">Done</strong> */}
-              <strong className="has-text-danger">Planned</strong>
+              <strong className={cn({"has-text-danger": !selectedTodo.completed})}>
```

Comment:
Fix this linter error
![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/54860641/acb74686-4fb7-4778-abbd-ec74eca86ab7)